docs/build-the-docs.md

```diff
@@ -15,7 +15,7 @@
 Before you can build the ownCloud documentation, you need to install the following software:
 
 - [git][link-git] (command: `git`)
-- [Node 8][link-Node-8] (command: `node`)
+- [Node][link-Node-8] (command: `node`)
```

Comment:
Please also fix the `[link-Node-8]` palceholder (including the definition on top of the page) to  `[link-Node]` to remove all remnants.

Reply:
This was fixed in another PR. After a rebase, this issue will go away.